src/nvim/buffer.c

```diff
@@ -2785,8 +2785,8 @@ void free_titles(void)
  * If maxwidth is not zero, the string will be filled at any middle marker
  * or truncated if too long, fillchar is used for all whitespace.
  */
-int 
-build_stl_str_hl (
+int
+build_stl_str_hl(
```

Comment:
I think that since you already touch this you should use style from the style guide:

```C
int build_stl_str_hl(
```